MapifyEditor/Export/Validators/Project/MapInfoValidator.cs

```diff
@@ -54,6 +63,18 @@ protected override IEnumerator<Result> Validate(Scenes scenes)
                     yield return Result.Warning($"MapInfo: '{nameof(MapInfo.fixedMapImage)}' should be square or it will be stretched. Current dimensions: {mapInfo.fixedMapImage.width}x{mapInfo.fixedMapImage.height}", mapInfo);
                 }
             }
+
+            if (mapInfo.LoadingScreenLogo != null)
+            {
+                var logo = mapInfo.LoadingScreenLogo;
+                const int optimalWidth = 1024;
+                const int optimalHeight = 575;
```

Comment:
Where are these numbers from? They seem extremely low. Considering that the aspect ratio is the only thing that matters, that's what should be checked (unless I'm wrong?)

Reply:
They are the resolution of the original DV loading screen logo. But checking aspect ratio is indeed better.

Reply:
Ah I thought this was for the background, that's why I was sus of it being so low. But yeah, aspect ratio is definitely better.